Unparsable values when databinding (#14287)
aspnet/AspNetCore.Docs: aspnetcore/blazor/components.md

Unlike `onchange`, which fires when the element loses focus, `oninput` fires when the value of the text box changes.

**Unparsable values**

> When a user provides an unparsable value to a databound element, the unparsable value is automatically reverted by default for the `onchange` event (`@bind="{PROPERTY OR FIELD}"`).

Reviewer: ```suggestion
When a user provides an unparsable value to a databound element, the unparsable value is automatically reverted to its previous value when the bind event is triggered.
```
